lib/plaid/products/item.rb

```diff
@@ -1,21 +1,25 @@
 module Plaid
   # Public: Class used to call the AccessToken sub-product.
-  class AccessToken
-    def initialize(client)
-      @client = client
-    end
-
+  class AccessToken < BaseProduct
     # Public: Rotate your access_token, keeping it attached to the item
     #
-    # Does a POST /item/access_token/invalidate call which will give you a new access_token
-    # allowing you to rotate access_tokens
+    # Does a POST /item/access_token/invalidate call which will give you a new
+    # access_token allowing you to rotate access_tokens.
     #
-    # access_token - access_token to invalidate and rotate
+    # access_token - The String access_token to invalidate and rotate.
     #
-    # Returns a parsed JSON with the new access_token and request id
+    # Returns an InvalidateResponse object with the new access_token and
+    # request id.
     def invalidate(access_token)
-      payload = { access_token: access_token }
-      @client.post_with_auth('item/access_token/invalidate', payload)
+      post_with_auth 'item/access_token/invalidate',
+                     InvalidateResponse,
+                     access_token: access_token
```

Comment:
Nit: would be nice to be consistent with parens around method calls (we do add parens elsewhere)

Reply:
I went ahead and removed all the parens surrounding post_with_auth :)